foundation/org.eclipse.persistence.oracle/src/main/java/org/eclipse/persistence/platform/database/oracle/Oracle21Platform.java

```diff
@@ -34,15 +33,11 @@
  */
 public class Oracle21Platform extends Oracle19Platform {
 
-    // Oracle JSON factory.
-    private final OracleJsonFactory factory;
```

Comment:
@Tomas-Kraus should know whether this needs to be marked `transient` or not instead